<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 31, 49);</string>
  </property>
  <widget class="QPlainTextEdit" name="plaintext_field">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ciphertext_field">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="key_field">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
  </widget>
  <widget class="QLabel" name="plaintext_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string>Plaintext:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ciphertext_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string>Ciphertext:</string>
   </property>
  </widget>
  <widget class="QLabel" name="key_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string>Key:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encrypt_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decrypt_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="cipher_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>801</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tibetan Machine Uni</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(28, 113, 216);</string>
   </property>
   <property name="text">
    <string>Cipher Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
